test/provider_devdax_memory.cpp

```diff
@@ -233,6 +233,13 @@ TEST_P(umfProviderTest, purge_force) {
     test_alloc_free_success(provider.get(), page_size, 0, PURGE_FORCE);
 }
 
+TEST_P(umfProviderTest, purge_force_unalligned_alloc) {
```

Comment:
Isn't it that devdax malloc is always aligned?
https://github.com/oneapi-src/unified-memory-framework/blob/a08a1218b09573dfe8ad25ee8d56b97b1743f4cd/src/provider/provider_devdax_memory.c#L281-L284